src/mc.rs

```diff
@@ -1230,3 +1217,1063 @@ pub unsafe fn resize_rust<BD: BitDepth>(
+        *mut DynPixel,
+        ptrdiff_t,
+        *const DynPixel,
+        ptrdiff_t,
+        *const int16_t,
+        libc::c_int,
+        libc::c_int,
+        libc::c_int,
+    ) -> (),
+>;
+pub type mct_fn = Option<
+    unsafe extern "C" fn(
+        *mut int16_t,
+        *const DynPixel,
+        ptrdiff_t,
+        libc::c_int,
+        libc::c_int,
+        libc::c_int,
+        libc::c_int,
+        libc::c_int,
+    ) -> (),
+>;
+pub type mct_scaled_fn = Option<
+    unsafe extern "C" fn(
+        *mut int16_t,
+        *const DynPixel,
+        ptrdiff_t,
+        libc::c_int,
+        libc::c_int,
+        libc::c_int,
+        libc::c_int,
+        libc::c_int,
+        libc::c_int,
+        libc::c_int,
+    ) -> (),
+>;
+pub type warp8x8t_fn = Option<
+    unsafe extern "C" fn(
+        *mut int16_t,
+        ptrdiff_t,
+        *const DynPixel,
+        ptrdiff_t,
+        *const int16_t,
+        libc::c_int,
+        libc::c_int,
+        libc::c_int,
+    ) -> (),
+>;
+pub type avg_fn = Option<
+    unsafe extern "C" fn(
+        *mut DynPixel,
+        ptrdiff_t,
+        *const int16_t,
+        *const int16_t,
+        libc::c_int,
+        libc::c_int,
+        libc::c_int,
+    ) -> (),
+>;
+pub type w_avg_fn = Option<
+    unsafe extern "C" fn(
+        *mut DynPixel,
+        ptrdiff_t,
+        *const int16_t,
+        *const int16_t,
+        libc::c_int,
+        libc::c_int,
+        libc::c_int,
+        libc::c_int,
+    ) -> (),
+>;
+pub type mask_fn = Option<
+    unsafe extern "C" fn(
+        *mut DynPixel,
+        ptrdiff_t,
+        *const int16_t,
+        *const int16_t,
+        libc::c_int,
+        libc::c_int,
+        *const uint8_t,
+        libc::c_int,
+    ) -> (),
+>;
+pub type w_mask_fn = Option<
+    unsafe extern "C" fn(
+        *mut DynPixel,
+        ptrdiff_t,
+        *const int16_t,
+        *const int16_t,
+        libc::c_int,
+        libc::c_int,
+        *mut uint8_t,
+        libc::c_int,
+        libc::c_int,
+    ) -> (),
+>;
+pub type blend_fn = Option<
+    unsafe extern "C" fn(
+        *mut DynPixel,
+        ptrdiff_t,
+        *const DynPixel,
+        libc::c_int,
+        libc::c_int,
+        *const uint8_t,
+    ) -> (),
+>;
+pub type blend_dir_fn = Option<
+    unsafe extern "C" fn(*mut DynPixel, ptrdiff_t, *const DynPixel, libc::c_int, libc::c_int) -> (),
+>;
+pub type emu_edge_fn = Option<
+    unsafe extern "C" fn(
+        intptr_t,
+        intptr_t,
+        intptr_t,
+        intptr_t,
+        intptr_t,
+        intptr_t,
+        *mut DynPixel,
+        ptrdiff_t,
+        *const DynPixel,
+        ptrdiff_t,
+    ) -> (),
+>;
+pub type resize_fn = Option<
+    unsafe extern "C" fn(
+        *mut DynPixel,
+        ptrdiff_t,
+        *const DynPixel,
+        ptrdiff_t,
+        libc::c_int,
+        libc::c_int,
+        libc::c_int,
+        libc::c_int,
+        libc::c_int,
+        libc::c_int,
+    ) -> (),
+>;
+#[derive(Copy, Clone)]
```

Comment:
Depending on if this PR or #417 lands first, we should get rid of this.

Reply:
#417 and #418 merged so could you remove this and any other warnings that show now?

Reply:
Done.